.github/workflows/compliance.yml

```diff
@@ -0,0 +1,34 @@
+name: Compliance
+
+on:
+  push:
+    branches: [ main ]
+  pull_request: {}
+
+permissions:
+  # https://docs.github.com/en/rest/overview/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-contents
+  contents: read
+
+jobs:
+  go:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-go@v4
```

Comment:
As I just merged #141, please also update this to `actions/setup-go@v4` (otherwise, we'd just get another dependabot PR soon).